doc/examples.rst

```diff
@@ -406,7 +406,7 @@ how deep the part is
 .. cadquery::
 
     result = cq.Workplane("front").box(3,2, 0.5)                 #make a basic prism
-    result = result.faces(">Z").vertices("<XY").workplane()  #select the lower left vertex and make a workplane
+    result = result.faces(">Z").vertices("<XY").workplane(centerOption="CenterOfMass")  #select the lower left vertex and make a workplane
```

Comment:
Is specifying `CenterOfMass` necessary in all the examples and tests? Normally the `ProjectedOrigin` and `CenterOrMass` options yield nearly identical results until material is removed in a non-symmetrical way.

Reply:
I only changed if an error occurred.